LOGGING/test/ILogger.Tests/ILoggerIntegrationTests.cs

```diff
@@ -121,6 +121,16 @@ public void ApplicationInsightsLoggerLogsExceptionAsTraceWhenSwitchIsFalse()
             ILogger<ILoggerIntegrationTests> testLogger = serviceProvider.GetRequiredService<ILogger<ILoggerIntegrationTests>>();
 
             testLogger.LogInformation("Testing");
+
+            try
+            {
+                ThrowException();
+            }
+            catch (Exception ex)
+            {
+                testLogger.LogError(ex, "LoggerMessage");
```

Comment:
nothing is doing an assert on this telemetry?

Reply:
I can do small assert but having full stack trace will be susceptible changes as code moves around.

Reply:
The following can be added ? I think it won't be affected unless the ThrowException() method is renamed ?
```
Assert.IsTrue((itemsReceived[2] as TraceTelemetry).Properties["ExceptionMessage"].Contains("StackTraceEnabled"));
Assert.IsTrue((itemsReceived[2] as TraceTelemetry).Properties["ExceptionStackTrace"].Contains("ThrowException"));
```

Reply:
I have added a test however it will need to be updated if code is moved around.